SOURCES/message-switch.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Xapi toolstack store and forward message switch
+After=network-online.target
+Wants=network-online.target
```

Comment:
Does message-switch really need network access? Also should there be a local filesystem dependency here?